Fix github url strings (org edx -> openedx)

sarina opened the pull request:
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

sarina committed:
fix: fix github url strings (org edx -> openedx)

sarina committed:
fix: update path to .github workflows to read from openedx org

kdmccormick commented on requirements/edx/github.in:
```diff
@@ -17,7 +17,7 @@
 #
 # For example:
 #
-#   git+https://github.com/edx/edx-lint.git@v0.3.2#egg=edx_lint==0.3.2
```
@sarina I've got a [PR in the works](https://github.com/openedx/edx-platform/pull/30719) that will update all the links in this file and regenerate the `.txt` files, plus some other changes. To spare me the merge conflicts do you mind removing the changes to `./requirements` from this PR?

sarina replied:
It would probably be easier for me to close the PR, and regenerate it when you're done.

kdmccormick replied:
Nevermind then, I would rather handle the merge conflicts than review this PR again 😅

sarina replied:
oh, just remove anything in the requirements dir? that's easy

kdmccormick replied:
awesome, thanks 👍🏻

sarina replied:
done! now to wait and see how many tests I need to fix 🤪

kdmccormick commented on pylintrc_backup:
```diff
@@ -0,0 +1,508 @@
+# ***************************
```
Did you create this file or did it come from edx_lint?

sarina replied:
You know it comes from edx-lint but I think at least half of the repos have this in their .gitignore or something because it never appeared in my testing. I can remove it.

kdmccormick replied:
Thanks. This exercise has been a great way to uncover all the different tooling quirks that different repos have lol

kdmccormick commented on openedx/features/name_affirmation_api/utils.py:
```diff
@@ -1,6 +1,6 @@
 """
 Utility functions for integration with Name Affirmation plugin
-(https://github.com/edx/edx-name-affirmation)
+(https://github.com/openedx/edx-name-affirmation)
```
```suggestion
(https://github.com/edx/edx-name-affirmation)
```

kdmccormick commented on .annotation_safe_list.yml:
```diff
@@ -127,7 +127,7 @@ djcelery.TaskState:
   ".. no_pii:": "No PII"
 
-# Via edx-oauth2-provider https://github.com/edx/edx-oauth2-provider
+# Via edx-oauth2-provider https://github.com/openedx/edx-oauth2-provider
```
```suggestion
# Via edx-oauth2-provider https://github.com/edx/edx-oauth2-provider
```

kdmccormick commented on .annotation_safe_list.yml:
```diff
@@ -99,7 +99,7 @@ django_notify.Settings:
   ".. no_pii:": "No PII"
 
-# Via django-openid-auth https://github.com/edx/django-openid-auth
+# Via django-openid-auth https://github.com/openedx/django-openid-auth
```
```suggestion
# Via django-openid-auth https://github.com/edx/django-openid-auth
```

kdmccormick commented on .editorconfig:
```diff
@@ -67,7 +64,7 @@
 #
 # ------------------------------
-# Generated by edx-lint version: 1.6
+# Generated by edx-lint version: 5.2.5
```
😮

sarina replied:
😱

nedbat replied:
!!!!!!

kdmccormick commented on common/static/common/js/jasmine.common.conf.js:
```diff
@@ -4,7 +4,7 @@
 jasmine.getFixtures().fixturesPath = '/base/';
 
-// https://github.com/edx/js-test-tool/blob/master/js_test_tool/templates/jasmine_test_runner.html#L10
+// https://github.com/openedx/js-test-tool/blob/master/js_test_tool/templates/jasmine_test_runner.html#L10
```
```suggestion
// https://github.com/edx/js-test-tool/blob/master/js_test_tool/templates/jasmine_test_runner.html#L10
```

kdmccormick commented on lms/envs/common.py:
```diff
@@ -965,7 +965,7 @@
     # .. toggle_default: False
     # .. toggle_description: Whether to display honor code agreement for learners before their first grade assignment
-    # (https://github.com/edx/edx-name-affirmation)
+    # (https://github.com/openedx/edx-name-affirmation)
```
```suggestion
    # (https://github.com/edx/edx-name-affirmation)
```

nedbat replied:
@sarina you need to take this suggestion.

sarina replied:
ack, missed that one!

kdmccormick commented on lms/static/sass/course/ccx_coach/_dashboard.scss:
```diff
@@ -63,7 +63,7 @@ table.ccx-schedule {
 form.ccx-form {
   line-height: 1.5;
-  // inspiration was taken from https://github.com/edx/ux-pattern-library
+  // inspiration was taken from https://github.com/openedx/ux-pattern-library
```
```suggestion
  // inspiration was taken from https://github.com/edx/ux-pattern-library
```

kdmccormick commented on lms/static/sass/shared/_alerts_pattern_library_shim.scss:
```diff
@@ -1,7 +1,7 @@
 // This file contains "I want alerts that look like the pattern library" sass
 // and should be replaced when moving to the patter library proper
-// https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_alerts.scss
+// https://github.com/openedx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_alerts.scss
```
```suggestion
// https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_alerts.scss
```

kdmccormick commented on openedx/core/djangoapps/oauth_dispatch/docs/decisions/0008-use-asymmetric-jwts.rst:
```diff
@@ -23,14 +23,14 @@ But to verify that the JWT was created by the OAuth Provider, the client should
 allow a Client to forward a verifiable token to other Clients.
 
-.. _ID Tokens are HMACed with their own client_secret: https://github.com/edx/edx-oauth2-provider/blob/7e59e30ae0bfd9eac4d05469768d79c50a90aeb7/edx_oauth2_provider/views.py#L155-L163
+.. _ID Tokens are HMACed with their own client_secret: https://github.com/openedx/edx-oauth2-provider/blob/7e59e30ae0bfd9eac4d05469768d79c50a90aeb7/edx_oauth2_provider/views.py#L155-L163
```
```suggestion
.. _ID Tokens are HMACed with their own client_secret: https://github.com/edx/edx-oauth2-provider/blob/7e59e30ae0bfd9eac4d05469768d79c50a90aeb7/edx_oauth2_provider/views.py#L155-L163
```

kdmccormick commented on .annotation_safe_list.yml:
```diff
@@ -161,7 +161,7 @@ milestones.MilestoneRelationshipType:
   ".. no_pii:": "No PII"
 
-# Via Django OAuth2 Provider https://github.com/edx/django-oauth2-provider
+# Via Django OAuth2 Provider https://github.com/openedx/django-oauth2-provider
```
```suggestion
# Via Django OAuth2 Provider https://github.com/edx/django-oauth2-provider
```

kdmccormick commented on scripts/provision-demo-course.sh:
```diff
@@ -23,7 +23,7 @@ mkdir -p /edx/app/demo
 	cd /edx/app/demo &&
 	rm -rf edx-demo-course &&
-	git clone https://github.com/edx/edx-demo-course.git --depth 1
+	git clone https://github.com/openedx/edx-demo-course.git --depth 1
```
```suggestion
	git clone https://github.com/openedx/openedx-demo-course.git --depth 1
```

kdmccormick commented on lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py:
```diff
@@ -21,22 +21,22 @@
     'help_tokens': 'https://github.com/openedx/help-tokens',
     'integrated_channels': 'https://github.com/openedx/edx-enterprise',
-    'learner_pathway_progress': 'https://github.com/edx/learner-pathway-progress',
+    'learner_pathway_progress': 'https://github.com/openedx/learner-pathway-progress',
```
```suggestion
    'learner_pathway_progress': 'https://github.com/edx/learner-pathway-progress',
```

kdmccormick commented on lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py:
```diff
@@ -21,22 +21,22 @@
+    'learner_pathway_progress': 'https://github.com/openedx/learner-pathway-progress',
     'lti_consumer': 'https://github.com/openedx/xblock-lti-consumer',
-    'notices': 'https://github.com/edx/platform-plugin-notices',
+    'notices': 'https://github.com/openedx/platform-plugin-notices',
```
```suggestion
    'notices': 'https://github.com/edx/platform-plugin-notices',
```

kdmccormick commented on lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py:
```diff
@@ -21,22 +21,22 @@
     'consent': 'https://github.com/openedx/edx-enterprise',
     'csrf': 'https://github.com/openedx/edx-drf-extensions',
-    'edx_name_affirmation': 'https://github.com/edx/edx-name-affirmation',
+    'edx_name_affirmation': 'https://github.com/openedx/edx-name-affirmation',
```
```suggestion
    'edx_name_affirmation': 'https://github.com/edx/edx-name-affirmation',
```

kdmccormick commented on lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py:
```diff
@@ -21,22 +21,22 @@
 EDX_REPO_APPS = {
     'bulk_grades': 'https://github.com/openedx/edx-bulk-grades',
-    'coaching': 'https://github.com/edx/platform-plugin-coaching',
+    'coaching': 'https://github.com/openedx/platform-plugin-coaching',
```
```suggestion
    'coaching': 'https://github.com/edx/platform-plugin-coaching',
```

sarina committed:
fix: update repo paths that stayed in the edx org

Co-authored-by: Kyle McCormick <kdmc@pm.me>

sarina commented:
@nedbat we need a strategy for dealing with pylint error R0201. Or should it be ignored? https://github.com/openedx/edx-platform/actions/runs/3040830796/jobs/4897322776 - there's around 300 violations (from just one of the builds)

kdmccormick commented:
@sarina hm, that violation seems like it'd be too complicated of a code modification to do automatically. Short of taking on a pile of refactoring work, I think you could either:
* apply [lint amnesty](https://github.com/openedx/edx-lint#using-lint-amnesty) if you think the warnings should eventually be fixed
* ignore it in pylintrc_tweaks if you think the warnings aren't ever worth fixing

I slightly lean ignoring the errors. In test modules, a lot of test cases are written as methods (instead of top-level functions) by convention even though they don't use `self`. Unless we're about to do a 180 that convention then I think this pylint violation is more noisy than it's worth.

If you could figure out a way to ignore the violation *only* for test modules, and then fix the remaining 10% of violations by hand, then that would be the best of both worlds. But I don't know offhand how to do that.

sarina commented:
I've been talking with @nedbat about this and I believe he thinks these should be fixed. I think they should be ignored - I've been trying to fix them, the easiest way is to use the `@staticmethod` decorator but that brings a host of new problems... a litany of errors like `TypeError: could not get code object for <staticmethod object at ...>`

For a few repos honestly I've just reverted the pylintrc upgrade and created a ticket for maintainers to take care of it. 🤷🏻‍♀️ 

If I can ignore it, great! that would make sense to me as I am not sure I understand why applying a decorator everywhere is better for the codebase.

nedbat commented:
I agree they should be ignored.  Even in product code, it's a bit persnickety to insist that `self` be used.  Let's add a disable to the tweaks file.

sarina commented:
@kdmccormick Are your requested changes met?

sarina commented:
tests are all passing except one unit test that popped up after i rebased. I'll work on addressing that once the pr is approved.

nedbat commented on openedx/features/name_affirmation_api/README.rst:
```diff
@@ -3,4 +3,4 @@ Name Affirmation API
 This directory contains a passthrough for the edx-name-affirmation plugin,
 in order to enable support in other plugins/packages such as edx-proctoring.
-See here: `https://github.com/edx/edx-name-affirmation`_.
+See here: `https://github.com/openedx/edx-name-affirmation`_.
```
```suggestion
See here: `https://github.com/edx/edx-name-affirmation`_.
```

nedbat reviewed (changes requested):
Two name-affirmation URLs to fix, and then good to go.

sarina commented:
@nedbat @kdmccormick could you review your requested changes?

Every time I rebase I get a few conflicts, so I'd appreciate being able to merge this 👍🏻

nedbat commented on cms/envs/common.py:
```diff
@@ -485,7 +485,7 @@
     # .. toggle_description: Whether to replace ID verification course/certificate requirement
     # with an in-course Honor Code agreement
-    # (https://github.com/edx/edx-name-affirmation)
+    # (https://github.com/openedx/edx-name-affirmation)
```
This is still wrong.
```suggestion
    # (https://github.com/edx/edx-name-affirmation)
```

sarina replied:
huh, I wonder why that's not going through, I definitely committed it via the UI. I will do it locally.

sarina commented:
Double checked line 488 from cms/envs/common.py in this diff and it is now resolved.

sarina committed:
chore: run Manage local config files from masters in edx_lint.
version 5.3.0
Commands:

    write FILENAME
        Write a local copy of FILENAME using FILENAME_tweaks for local tweaks.

    check [FILENAME ...]
        Check that FILENAME has not been edited since writing.
        If FILENAME is omitted, all existing files are checked.

    list
        List the files that edx_lint can provide.

    update
        Re-write any edx_lint-written files that exists on disk.
     update with the current version of the repo.

kdmccormick reviewed (approved):
🚀
